x-pack/plugins/fleet/server/services/agents/checkin/state_new_actions.ts

```diff
@@ -167,8 +174,14 @@ export async function createAgentActionFromPolicyAction(
     }
   );
 
+  const permissions =
+    'policy' in newAgentAction.data
+      ? newAgentAction.data.policy.outputPermissions.default
+      : newAgentAction.data.config.outputPermissions.default; // agent <= 7.9
```

Comment:
should these be `output_permissions.default`?

Reply:
ah, yes. I'm surprised typescript didn't catch it